apple/RNCWebView.m

```diff
@@ -324,6 +325,10 @@ - (void)removeFromSuperview
 -(void)showFullScreenVideoStatusBars
 {
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
+    if (!_autoManageStatusBarEnabled) {
+      return;
+    }
+
     _isFullScreenVideoOpen = YES;
     RCTUnsafeExecuteOnMainQueueSync(^{
       [RCTSharedApplication() setStatusBarStyle:UIStatusBarStyleLightContent animated:YES];
```

Comment:
I know you didn't write this code, but while we're in here, we shouldn't explicitly set the `LightContent` here. Instead, use the `self->_savedStatusBarStyle` property (like in the `hide` method below).

```suggestion
      [RCTSharedApplication() setStatusBarStyle:self->_savedStatusBarStyle animated:YES];
```